presto-docs/src/main/sphinx/release/release-0.231.rst

```diff
@@ -0,0 +1,53 @@
+=============
+Release 0.231
+=============
+
+General Changes
+_______________
+* Add support for get partition names by filter.
+* Add forms of approx_percentile accepting an accuracy parameter.
+* Improve error handling for geometry deserialization edge cases.
+* Fix failures caused by invalid plans for queries with repeated lambda expressions in the order by clause.
```

Comment:
According to release note guideline, fixes should go first: https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines#order-of-changes

Reply:
yes changed the order of sections and changes according to the wiki